bug_change_status_page.php

```diff
@@ -158,7 +159,7 @@
 
 <?php
 	$t_current_resolution = $t_bug->resolution;
-	$t_bug_is_open = $t_current_resolution < $t_resolved;
+	$t_bug_is_open = $t_current_resolution < $t_resolution_fixed;
```

Comment:
_$t_bug_is_open_ is somewhat confusing, as the notion of "open" usually refers to the issue's _status_ (i.e. open == status < bug_resolved_status_threshold)

I suggest renaming _$t_bug_is_open_ to **$t_bug_resolution_is_fixed** or something similar, to avoid confusion.

Furthermore, considering the default value for $g_resolution_enum_string, the `<` operator could be incorrect, since fixed = 20, but I would also consider at least 30:reopened (and 80:suspended as well) as "open", so my expectation would be that closing a _reopened_ issue would mark it as _fixed_. Although you could argue that this is a separate issue.